autoptsclient_common.py

```diff
@@ -548,11 +549,26 @@ def wrapper(*args):
 
         end_time = str(round(datetime.timedelta(
             seconds=end_time).total_seconds(), 3))
-
-        print("{}".format(test_case.status).ljust(16) +
-              end_time.rjust(len(end_time)) +
-              retries_msg.rjust(len("#{}".format(retries_max)) + margin) +
-              regression_msg.rjust(len("REGRESSION") + margin))
+        if test_case.status == "PASS":
+            print(colored(("{}".format(test_case.status).ljust(16) +
+                end_time.rjust(len(end_time)) +
+                retries_msg.rjust(len("#{}".format(retries_max)) + margin) +
+                regression_msg.rjust(len("REGRESSION") + margin)), "green"))
+        elif test_case.status == "FAIL":
```

Comment:
Colored text should not be printed if stdout is redirected into a file, so that file won't contain ANSI escape codes. Viewing text with escape codes in a text editor shows:

^[[32mauto-pts is awesome!^[[0m

Solution could be not to use colors if stdout is not a TTY:

```
txt = 'auto-pts is awesome!'
if sys.stdout.isatty():
    txt = colored(txt, 'green')
print txt
```

Reply:
Done